tests/Integ/SmokeContext.php

```diff
@@ -0,0 +1,363 @@
+<?php
+
+namespace Aws\Test\Integ;
+
+use Aws\Exception\AwsException;
+use Aws\JsonCompiler;
+use Aws\Result;
+use Aws\Sdk;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use JmesPath\Env;
+use PHPUnit_Framework_Assert;
+
+class SmokeContext extends PHPUnit_Framework_Assert implements
+    Context,
+    SnippetAcceptingContext
+{
+    protected static $configDefaults = [
+        'version' => 'latest',
+        'region' => 'us-east-1',
+        'profile' => 'integ',
+    ];
+
+    protected static $services = [
```

Comment:
The tags on the feature files aren't 1-to-1 matches to what's in the manifest file. I removed the exact matches from the latest commit but there's still a sizable config array.